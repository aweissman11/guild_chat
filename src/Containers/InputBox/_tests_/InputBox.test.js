import React from 'react';
import { shallow } from 'enzyme';

import { InputBox } from '../';

describe('InputBox', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<InputBox />);

    expect(wrapper).toMatchSnapshot();
  });
});