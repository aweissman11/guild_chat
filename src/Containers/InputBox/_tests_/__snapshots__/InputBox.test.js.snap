// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputBox should match snapshot 1`] = `ShallowWrapper {}`;
